src/feature/common/types/ParamValueType.ts

```diff
@@ -0,0 +1,10 @@
+export enum ParamValueType {
```

Comment:
Generally we prefer string union over Enum in TS

Reply:
Got it